cordyceps/src/list.rs

```diff
@@ -897,6 +913,17 @@ impl<T: Linked<Links<T>> + ?Sized> List<T> {
         }
     }
 
+    /// Returns an iterator over the items in this list, by pointer.
+    #[must_use]
+    pub fn raw_iter(&self) -> RawIter<'_, T> {
```

Comment:
also, turbo nit: i might consider calling this `iter_raw` rather than `raw_iter`, which feels more consistent with e.g. `iter_mut`

Reply:
I'm cool with either, I went back and forth.